### PR TITLE
improved the ids

### PR DESCRIPTION
**Pull request by azabeli**

### CODE REVIEW

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
These are a good start but I think it would be good to be a bit more descriptive in the id names. For something like renaming a table it might be good to name the id based off of whatever information that table is showing.

File: app/templates/events/trackVolunteers.html [hunk 1]
```diff
@@ -236,7 +236,7 @@ <h4>RSVP Waitlist</h4>
     <div class="modal-content">
       <div class="modal-header">
         {% set modalTitle = "Add Volunteer to Waitlist" if rsvpFull else("Add Volunteer RSVP" if rsvpNeeded else "Add Volunteer") %}
-        <h5 class="modal-title" id="exampleModalLabel">{{modalTitle}}</h5>
+        <h5 class="modal-title" id="ModalLabel">{{modalTitle}}</h5>
```
Comment (BrianRamsay) [inline app/templates/events/trackVolunteers.html]: If the id is never used, it doesn't need to be there at all.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
We should review more html than just the examples that Zach provided.

**Comment by AndersonStettner** [comment]
[Improve Id Usage #564](https://trello.com/c/cDs9GSJ6/430-improve-id-usage-564)

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
Resolve the merge conflicts.

**Review by BrianRamsay — DISMISSED** [review]

**Pull request merged**